Documentation/contributing/release/stable.rst

```diff
@@ -148,6 +148,25 @@ If you intent to release a new feature release, see the
       ./prepare_artifacts.sh /path/to/cilium/repository/checked/out/to/release/commit
       git push
 
+#. Prepare Helm changes using the `Cilium Helm Charts Repository <https://github.com/cilium/charts/>`_
+   for the vX.Y helm charts, and push the changes into that repository (not the main cilium repository):
+
+   In the ``cilium/cilium`` repository:
+
+   #. ``git checkout vx.y -b vx.z-dev``
+   #. Change the ``VERSION`` file to ``x.y-dev``
+   #. Run ``make -C install/kubernetes``
+
+   In the ``cilium/charts`` repository:
+
+   ::
+
+      ./prepare_artifacts.sh /path/to/cilium/repository/checked/out/to/release/commit
+      git push
```

Comment:
I note that whenever we change the helm charts on a release branch, we should probably run this again based on the latest version of the branch. I wonder even whether it's worth adding to the regular bugfix release steps.

Reply:
I'm confused, AFAIK this is the guide used for the patch releases.

Reply:
Yes, and patch releases can update the helm charts, in which case the `1.8-dev` helm charts would start to diverge from `1.8.1` and so on. We won't automatically pick up those fixes into the 1.8-dev helm chart.

Reply:
Misunderstanding on my part.

We have the following guides:

* General / micro / stable / bugfix
* Feature / minor
* RC

I got mixed up between them since we have different terminology. This is the right place.